bitsv/network/services/network.py

```diff
@@ -14,22 +19,33 @@ class NetworkAPI:
     """
     A Class for handling network API redundancy.
 
-    FIXME network API redundancy and RegTesting / Testnet - see Issue section on github"""
+    :param apis: A dictionary of {api_name : api_keys} in order of preference for redundancy. (Python 3.6 dicts ordered)
+    :type apis: ``dict`` of ``str`` : ``str``
+    :param network: 'main', 'test' or 'stn' --> feeds into the bitsv.network.NetworkAPI class for redundancy
+    :type network: ``str``
+    """
 
     IGNORED_ERRORS = (ConnectionError,
                       requests.exceptions.ConnectionError,
                       requests.exceptions.Timeout,
                       requests.exceptions.ReadTimeout)
 
-    # BitIndex for all mainnet functions
-    GET_BALANCE_MAIN = [BitIndex.get_balance]
-    GET_TRANSACTIONS_MAIN = [BitIndex.get_txs]
-    GET_UNSPENT_MAIN = [BitIndex.get_utxo]
-    BROADCAST_TX_MAIN = [BitIndex.broadcast_rawtx]
-    GET_TX_MAIN = [BitIndex.get_tx]
+    def __init__(self, apis, network):
```

Comment:
Please correct me if I'm wrong, but it seems like setting an API has no effect? Just ends up bitindex anyways.

Reply:
Yes - right now it is. But the idea is to have the structure clearly laid out for contributors to know how to plug new apis in. E.g. when https://developers.whatsonchain.com/ finishes their api and also adding blockchair (tends to be one of the most robust apis under heavy load)...